docs/docs/connect/data-source/duckdb.md

```diff
@@ -22,7 +22,7 @@ There are several limitations with deployment to Rill Cloud, so we do not recomm
 
 ## Attaching an External DuckDB
 
-In the default `connectors/duckdb.yaml` file, you can add `init_sql` or `attach` statements to attach an external database to Rill's embedded DuckDB. The `attach` statement runs before `init_sql`, allowing you to attach your database and execute subsequent initialization SQL queries. For more details on the YAML configuration, see the [DuckDB reference page](/reference/project-files/connectors#duckdb).
+In the default `connectors/duckdb.yaml` file, you can add `init_sql` statement to attach an external database to Rill's embedded DuckDB.  For more details on the YAML configuration, see the [DuckDB reference page](/reference/project-files/connectors#duckdb).
```

Comment:
I think it'd be nice to say a one-liner here about what `init_sql` does generally (i.e. "executes a SQL statement during database initialization")